src/sql/connection/connection.py

```diff
@@ -205,12 +205,17 @@ def set(
                 existing = rough_dict_get(cls.connections, descriptor)
                 if existing and existing.alias == alias:
                     cls.current = existing
-                # passing an existing descriptor and not alias: use existing connection
                 elif existing and alias is None:
+                    if cls.current != existing:
+                        display.message(
+                            f"Switching to connection {alias or descriptor}"
```

Comment:
This line is repeated 2 times. can be decalred at one place

Reply:
This one is called when we switch from existing connection, the other is  called when a new connection is made

Reply:
I meant the prefix `Switching to connection ` can be declared at one place like 
`SWITCH_PREFIX=Switching to connection` and reused

Reply:
[Added!](https://github.com/ploomber/jupysql/pull/796/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R202)